imap_processing/tests/codice/test_codice_l1a.py

```diff
@@ -35,7 +35,7 @@
     (1, 128),  # lo-pha
 ]
 EXPECTED_ARRAY_SIZES = [
-    123,  # hskp
+    129,  # hskp
```

Comment:
Interesting, you got a few more packets now?

Reply:
Your comment here made me realize that the test for this is poorly named, because these values are actually the expected number of variables in the CDF (i.e. `len(dataset)`), not the data array sizes. I will update this in a new commit.